components/nodemanager-service/nodemanager.go

```diff
@@ -20,11 +22,15 @@ import (
 	"github.com/chef/automate/components/nodemanager-service/pgdb"
 	"github.com/chef/automate/components/nodemanager-service/polling"
 
+	"github.com/chef/automate/lib/cereal"
+	grpccereal "github.com/chef/automate/lib/cereal/grpc"
+	"github.com/chef/automate/lib/cereal/patterns"
 	"github.com/chef/automate/lib/grpc/health"
 	"github.com/chef/automate/lib/grpc/secureconn"
 	"github.com/chef/automate/lib/version"
 
 	"github.com/chef/automate/lib/tracing"
+	"github.com/sirupsen/logrus"
```

Comment:
[nit] I think we're trying to have imports grouped by stdlib, external, local, so this and some below should go up with `rrule-go`